config/prod/prow/config_knative.yaml

```diff
@@ -1280,74 +1280,80 @@ periodics:
       limits:
         memory: 16Gi
   - nightly: true
-    args:
-    - --publish
-    - --tag-release
-    - --release-gcs knative-gcp-nightly
-    - --release-gcr gcr.io/knative-gcp-nightly
+    args: [
```

Comment:
IMO, it would be better to keep the original style with dashes and not brackets because in the other file below (on other places) you're keeping the "dash" style.
```
args:
-
-
```

Reply:
The other file `config/prod/prow/jobs/config.yaml` is generated, so we have no control how it's written. As it's generated the readability of it makes less of a concern to me.

I did such change here to keep parameters that relates to each other on the same line. I think it's easier to understand in this way:

```yaml
args: [
  '--tag-release',
  '--release-gcs', 'knative-gcp-nightly'
]
```
instead of:

```yaml
args:
- --tag-release
- --release-gcs
- knative-gcp-nightly
```
In the second example, I think it's harder to understand that `knative-gcp-nightly` is a value of `--release-gcs` flag.